Code/test_hrm.py

```diff
@@ -1,3 +1,29 @@
+def test_initialization():
+    """
+    :Author: Tim Hoer
+    :Date: September 16, 2017
+    :Notes: Ensures ECG class initializes inputs correctly.
+    """
+    from ecg import ECG
+    test = ECG(file='ecg_data1.csv', window=20, brady_max=40, tachy_min=100)
+    assert(test.window == 20)
+    assert(test.threshold == (40, 100))
+
+
+def test_class_write():
+    """
+    :Author: Tim Hoer
+    :Date: September 16, 2017
+    :Notes: Ensures ECG class writes output file in correct output.
+    """
+    from ecg import ECG
+    import os
+    test = ECG(file='ecg_data1.csv', window=20, brady_max=40, tachy_min=100)
+    test.write_file()
+    assert(os.path.isfile('ecg_data1_out.txt'))
```

Comment:
Check out the use of temporary file fixtures for testing: https://docs.pytest.org/en/latest/tmpdir.html

Reply:
If you're interested in testing that the output of the file looks a certain way (and that people didn't check in code that breaks an expected text formatting inside the file) you could compare the file to a test file (but that file would have to be updated every time you change your format)

Reply:
(I see you did some file content checking in your `test_write_to_file` for the module based version, nice)

Reply:
Our write function writes to the location of the input file so  I don't think I can create a temporary directory to write my files from the test script - is module based content checking sufficient?